### ISSUE TITLE
Issues with JSON bracket matching

### ISSUE THREAD

**Issue by isc-tleavitt** [intersystems/isc-codetidy#26]
Particularly, CodeTidy doesn't seem to like definitions with objects within arrays - e.g.:
`set foo = {"array":[{"bar":"baz"}]}`

The last } ends up being indented as if it was the end of a method.

**Comment by isc-tleavitt** [comment]
Let's collapse [{ and ]} into a single line like https://jsonlint.com/ does.

**Comment by isc-tleavitt** [comment]
Example, for:
```
	set foo = {"array":[{"bar":"baz"}]}
```

Or:
```
	set foo = {
		"array": [
	{
			"bar":"baz"
		}
	]
	}
```

The output should be:
```
	set foo = {
		"array": [{
			"bar":"baz"
		}]
	}
```

**Comment by isc-tleavitt** [comment]
Closed via #30
